src/components/create.vue

```diff
@@ -214,21 +237,21 @@
             step3: async function() {
                 let apkey, mpkey, opkey;
                 if (this.$data.accountname == "") {
-                    this.$data.errorMsg = this.$t('missing_account_error',{'chain': 'BTS'});
+                    this.$data.errorMsg = this.$t('missing_account_error',{'chain': this.$data.selectedChain});
```

Comment:
Vue binds data directly on the instance. What is the difference to this (other than cleaner maybe?)?

`this.$data.selectedChain <> this.selectedChain`

Reply:
I'll be honest. I have NO idea why I'm using this.$data there....I was wondering the same when I added the selectedChain stuff seeing as that code (accountname, errormsg etc.) was written a while back.

I need to revisit to see why I did that and if no reason, update